### PR TITLE
Add benchmark for serializing headers

### PR DESCRIPTION
**Pull request by bdraco**
SSIA

### CODE REVIEW

File: tests/test_benchmarks_http_writer.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from multidict import CIMultiDict
+from pytest_codspeed import BenchmarkFixture
+
+from aiohttp import hdrs
+from aiohttp.http_writer import _serialize_headers
+
+
+def test_serialize_headers(
+    loop: asyncio.AbstractEventLoop, benchmark: BenchmarkFixture
```
Comment (bdraco) [inline tests/test_benchmarks_http_writer.py]:
```suggestion
    benchmark: BenchmarkFixture
```

File: tests/test_benchmarks_http_writer.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from multidict import CIMultiDict
+from pytest_codspeed import BenchmarkFixture
+
+from aiohttp import hdrs
+from aiohttp.http_writer import _serialize_headers
+
+
+def test_serialize_headers(
+    benchmark: BenchmarkFixture
+) -> None:
```
Comment (bdraco) [inline tests/test_benchmarks_http_writer.py]:
```suggestion
def test_serialize_headers(benchmark: BenchmarkFixture) -> None:
```

File: tests/test_benchmarks_http_writer.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+from multidict import CIMultiDict
+from pytest_codspeed import BenchmarkFixture
+
+from aiohttp import hdrs
+from aiohttp.http_writer import _serialize_headers
+
+
+def test_serialize_headers(benchmark: BenchmarkFixture) -> None:
+    """Benchmark 100 _serialize_headers."""
```
Comment (bdraco) [inline tests/test_benchmarks_http_writer.py]:
```suggestion
    """Benchmark 100 calls to _serialize_headers."""
```

**Review by asvetlov — APPROVED** [review]

File: tests/test_benchmarks_http_writer.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+"""codspeed benchmarks for http writer."""
+
+import asyncio
+
```
Comment (bdraco) [inline tests/test_benchmarks_http_writer.py]:
```suggestion
```

**Comment by bdraco** [comment]
thanks

File: tests/test_benchmarks_http_writer.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            hdrs.PRAGMA: "no-cache",
+            hdrs.EXPIRES: "0",
+            hdrs.LAST_MODIFIED: "Mon, 23 May 2005 22:38:34 GMT",
+            hdrs.ETAG: "1234567890",
+        }
+    )
+
+    @benchmark
+    def _run() -> None:
+        for _ in range(100):
```
Comment (Dreamsorcerer) [inline tests/test_benchmarks_http_writer.py:32]: Isn't the job of the benchmark to figure out number of iterations to run?

**Pull request merged**